.github/workflows/run-analysis.yml

```diff
@@ -12,14 +12,195 @@ jobs:
       - uses: actions/checkout@v3
         name: Checkout repo
 
+# Build docker container and download testing data
       - name: Build container and download data
         uses: ./
         id: download-data
         with:
           args: OPENPEDCAN_URL=https://s3.amazonaws.com/d3b-openaccess-us-east-1-prd-pbta/open-targets OPENPEDCAN_RELEASE=testing bash download-data.sh
 
-      - name: Run RUN-molecular-subtyping-integrate in container
+# Molecular subtyping modules
+      - name: Run Molecular Subtyping - MB
+        uses: ./
+        id: RUN-molecular-subtyping-MB
+        with:
+          args: bash analyses/molecular-subtyping-MB/run-molecular-subtyping-mb.sh
```

Comment:
With the way the action is setup, passing `bash` here is not necessary since it's already running from the entrypoint.

https://github.com/PediatricOpenTargets/OpenPedCan-analysis/blob/1eedaf5825fc999415ff5fd712f98a6f17718b92/scripts/run-analysis.sh#L3

Reply:
ok, thanks. I wasn't sure - in OpenPBTA, they are written either way and there didn't seem to be rhyme or reason/was mixed. Just removed 🤞